src/VirtoCommerce.Platform.Web/Controllers/Api/ModulesController.cs

```diff
@@ -84,6 +84,28 @@ public ActionResult<ModuleDescriptor[]> GetModules()
             return Ok(retVal);
         }
 
+        /// <summary>
+        /// Get installed modules with errors
+        /// </summary>
+        /// <returns></returns>
+        [HttpGet]
+        [Route("errors")]
+        [AllowAnonymous]
+        public ActionResult<ModuleDescriptor[]> GetModulesErrors()
```

Comment:
Better move this method to the `DiagnosticsController` and use `IModuleCatalog` instead of external

Reply:
Moved